Combat2.0 cloaking n shield n armor

1sourcecontrol opened the pull request:
Uploaded the shields and cloaking gear for C2.0.
Will add some extra hardpoint equipment, and salvage some environmental suits from Taylor's original stuff. (Btw, I skipped adding helmet armor because I don't think we need it.)
Trying to get the stuff donnnne so we can play in zeeh fall :) 👍 

Switched the costs from nanites to stamina. __EDIT__ changed from stamina back to nanites

1sourcecontrol committed:
hardpoint stuff

1sourcecontrol committed:
Added Shields and Cloaking Hardpoints

1sourcecontrol committed:
misc equipment

1sourcecontrol committed:
converted concealed rigs and environ suits to c2.0, tried to improve text

trowl223 commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -165,386 +203,284 @@ Cost For Arm or Leg Armor:	2900
+	
+	=====   LEVEL 10 REQUIREMENT   =====
+	
+---- C-Class Plasma Shield ----
```
Change name to "C-Class Self Sustained Nanite (SSN) Shield"

1sourcecontrol committed:
revert from stamina to nanites changed name of shield

bleehu commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -88,12 +71,67 @@ this by providing 'Hard Points'. Hard Points are special joints and fittings tha
+"Secondary Modules". After you purchase and install a Primary Module, you can also purchase
+and install a Secondary Module(s) that augments the benefits of the Primary Module. The 
+benefits are listed in the description of a given module. They do not mix and match,
+Secondary Modules only work for the Primary Module they are designed for.
```
I like this description of primary and secondary modules. I didn't know this was a thing, but after reading like 4 sentences, I feel like I get it.

bleehu commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -88,12 +71,67 @@ this by providing 'Hard Points'. Hard Points are special joints and fittings tha
+is reduced in RP. If the suit is hit more times than its "Suit Integrity"
+rating, it will no longer be able to maintain air pressure (player health 
+has to be damaged in order for "Suit Integrity" to count a hit). Suits 
+traditionally come in white, yellow or red.
```
I love that this is finally made explicit and I  feel that you struck a very good balance with acknowledging that if you shoot a space suit too many times, it stops protecting you from space while at the same time not making it too complex.

bleehu commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -88,12 +71,67 @@ this by providing 'Hard Points'. Hard Points are special joints and fittings tha
-reduces your movement speed by -1m per material added.
+Adding any armor materials to a chest, leg, or arm rig adds to
+your movement speed penalties by -1m per material added.
 
```
Should we maybe mention here that movement penalties can be remediated by stats? Or do you feel that is covered well somewhere else?

1sourcecontrol replied:
I think that's a good idea. The more we mention it, the easier it is to remember

bleehu commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -165,386 +203,284 @@ Cost For Arm or Leg Armor:	2900
+    =====================================
+Armor rigs cannot be worn in combination with armor
+other rigs, plate carriers, soft armour, or tactical vests.
+	
```
*Other armor rigs?

bleehu commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -165,386 +203,284 @@ Cost For Arm or Leg Armor:	2900
+	
+	
+*michael halp me plsz* I took the armor descriptions from
+1.0 armor and put it below:
```
Why keep it if you have more flavor text in the new stuff?

bleehu replied:
I see that most of the flava has been removed. I kinda like to keep flavor in a per-item basis anyway. nuke it.

1sourcecontrol replied:
I don't have much flavor text in the new stuff. But, for now, I think it's good to have some flavor text for each armor group (recon, light, medium, w/e) so that players understand some of the lore, that we aren't running around in halo armor

bleehu commented on play/PlayerRules/CharacterCreation/05_ArmourList.txt:
```diff
@@ -165,386 +203,284 @@ Cost For Arm or Leg Armor:	2900
+Prim Module Hard Pts: 	1
+Detect DC:		75
+Moving Detect DC:   	50
+Cloak Turn Bonus:	+0
```
What is cloak turn bonus? I don't see it in the basic rules?

1sourcecontrol replied:
ahhh, i'll remedy that. It's just a bonus to the number of turns you can stay cloaked at one time

bleehu reviewed (changes requested):
Lots of very good stuff here! Some minor tweaks and an explination or two and we should be good!

1sourcecontrol committed:
added description for the use of

1sourcecontrol committed:
changed flightsuits and slightly changed description for armor rigs

removed flightsuit's ability to work with other armor rigs and made them have an armor chest plate

1sourcecontrol committed:
prelim armor rigs

1sourcecontrol committed:
completed armor rigs

1sourcecontrol committed:
clarifying armor material purchasing for concealed and enviro suits
